src/mpid/ch4/src/ch4r_win.c

```diff
@@ -115,7 +115,7 @@ static void get_info_perf_preference_str(int val, char *buf, size_t maxlen)
 
     if (val & (1 << MPIDIG_RMA_LAT_PREFERRED)) {
         MPIR_Assert(c < maxlen);
-        c += snprintf(buf + c, maxlen - c, "%slat", (c > 0) ? "," : "");
+        c += snprintf(buf, maxlen, "lat");
```

Comment:
The assertion should be removed as well.

Reply:
Good catch. fixed